### PR TITLE
feat: Update to latest common, add polling interval parameter

### PR DESCRIPTION
**Pull request by typotter**
fixes FF-3485

### CODE REVIEW

File: docs/js-client-sdk.eppojsclient.getexperimentcontainerentry.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
```
Comment (typotter) [inline docs/js-client-sdk.eppojsclient.getexperimentcontainerentry.md:1]: The experiment container registry docs must have been missed in a previous commit?

File: docs/js-client-sdk.iclientconfig.pollaftersuccessfulinitialization.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## IClientConfig.pollAfterSuccessfulInitialization property
 
-Poll for new configurations (every 30 seconds) after successfully requesting the initial configuration. (default: false)
+Poll for new configurations (every `polingIntervalMs`<!-- -->) after successfully requesting the initial configuration. (default: false)
```
Comment (typotter) [inline docs/js-client-sdk.iclientconfig.pollaftersuccessfulinitialization.md:7]: There isn't a space here in the source code, so I'm not sure why it autodocs to this.

File: docs/js-client-sdk.eppojsclient.getexperimentcontainerentry.md (hunk 1 above)
Comment (aarsilv) [reply docs/js-client-sdk.eppojsclient.getexperimentcontainerentry.md:1]: 🤷‍♂️ 

I hate these doc generators 😞

**Review by leoromanovsky — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -59,6 +59,6 @@
     "webpack-cli": "^4.10.0"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.2.0"
+    "@eppo/js-client-sdk-common": "4.3.0"
```
Comment (aarsilv) [inline package.json:62]: 📈

**Review by aarsilv — APPROVED** [review]

**Pull request merged**